src/MCPClient/lib/clientScripts/validateDerivative.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import print_function
```

Comment:
Yes, assuming the clientScripts/lib/ approach is what we go with, the entire policyCheck.py could be moved there too and made non-executable. It's not really used as a client script on its own; it's always imported and subclassed just like validateDerivative.py